src/InvertedIndices.jl

```diff
@@ -183,12 +183,15 @@ end
 
 # This is an interesting need — we need this because otherwise indexing with a
 # single multidimensional boolean array ends up comparing a multidimensional cartesian
-# index to a linear index. Does this need addressing in Base, too?
+# index to a linear index.
 @inline Base.to_indices(A, I::Tuple{Not{<:NIdx{1}}}) = to_indices(A, (eachindex(IndexLinear(), A),), I)
 @inline Base.to_indices(A, I::Tuple{Not{<:NIdx}}) = to_indices(A, axes(A), I)
-# Arrays of Bool are even more confusing as they're sometimes linear and sometimes not
-@inline Base.to_indices(A, I::Tuple{Not{<:AbstractArray{Bool, 1}}}) = to_indices(A, (eachindex(IndexLinear(), A),), I)
-@inline Base.to_indices(A, I::Tuple{Not{<:Union{Array{Bool}, BitArray}}}) = to_indices(A, (eachindex(A),), I)
+if VERSION < v"1.11"
```

Comment:
```suggestion
if VERSION < v"1.11.0-DEV.1157"
```
This is the `contrib/commit-name.sh` output for the PR noted in the comment.